### PR TITLE
Feature/popular threads screen

### PR DESCRIPTION
**Pull request by zeroregard**
Maybe we can use power of two (2, 4, 8..) for margins/padding?
 
image

image

### CODE REVIEW

File: lib/helpers/api.dart [hunk 1]
```diff
@@ -202,25 +203,13 @@ class KnockoutAPI {
     }
   }
 
-  Future<List<SubforumThreadLatestPopular>> latestThreads() async {
-    final response = await _request(type: 'get', url: 'thread/latest');
-
+  Future<List<SignificantThread>> getSignificantThreads(
+      SignificantThreads threadsToFetch) async {
+    final endpoint = "v2/threads/${threadsToFetch.name.toLowerCase()}";
```
Comment (zeroregard) [inline lib/helpers/api.dart:208]: There's a version 2 out, maybe we should slowly convert each endpoint to this version before just adding `v2/` to the end of `KNOCKOUT_URL` ?

Comment (zeroregard) [reply lib/helpers/api.dart:208]: image
aha aha

Comment (dasmikko) [reply lib/helpers/api.dart:208]: We convert as we go. 👍

File: lib/widgets/drawer/mainDrawer.dart [hunk 2]
```diff
@@ -2,13 +2,24 @@ import 'package:flutter/material.dart';
...
 import 'package:knocky/screens/login.dart';
+import 'package:knocky/screens/significantThreads.dart';
 
 import 'drawerListTile.dart';
 
 class MainDrawer extends StatelessWidget {
   final AuthController authController = Get.put(AuthController());
 
+  navigateTo(BuildContext context, Widget screen) {
+    Navigator.push(context, MaterialPageRoute(builder: (context) => screen));
```
[the remark this answers is not in the document]
Comment (dasmikko) [reply lib/widgets/drawer/mainDrawer.dart:15]: Well, I was thinking that the drawer should only be on the main screen, and not on all screens.

**Pull request merged**